src/Microsoft.ML.TimeSeries/SrCnnEntireAnomalyDetector.cs

```diff
@@ -351,6 +360,15 @@ internal sealed class SrCnnEntireModeler
             private static readonly double _deanomalyThreshold = 0.35;
             private static readonly double _boundSensitivity = 93.0;
             private static readonly double _unitForZero = 0.3;
+            private static readonly double _minimumScore = 0.0;
+            private static readonly double _maximumScore = 1.0;
+            // If the score window is smaller than this value, the anomaly score is tend to be small.
```

Comment:
just curious,` score window` is the same as `batchSize`?

Reply:
It is not the same as batchSize, it is the `_judgementWindowSize`.

---
In reply to: [527820502](https://github.com/dotnet/machinelearning/pull/5502#discussion_r527820502) [](ancestors = 527820502)